src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -53,6 +55,17 @@ Can differ from `num_psis_draws`.
 - `num_elbo_draws` - Number of Monte Carlo draws to evaluate ELBO.
 Must be a positive integer.  Default value is $25$.
 
+- `psis_resample` - If True ($1$), perform psis resampling on samples returned
+from individual pathfinders. If False ($0$), returns all `num_paths` * `num_draws samples` draws
```

Comment:
False not capitalized here :-)

Reply:
Put the asterisk inside the quotes or pull it out into $\times$ if you want it to look like math.